File: Day5_reverseWordsInString/revWords.cs
using System;
class HelloWorld {
  static void Main() {
        string str = "Technology Encroyable";
		string[] strArr = str.Split(" ");
		
		string rev = "";
		for(int i = strArr.Length-1;i>=0;i--)
		{
		    rev += strArr[i] + " ";
		}
		Console.WriteLine(rev);
  }
}

File: Day5_reverseWordsInString/revWords.java
public class TechnologyEncroyable
{
	public static void main(String[] args) {
		String str = "Technology Encroyable";
		String[] strArr = str.split(" ");
		
		String rev = "";
		for(int i = strArr.length-1;i>=0;i--){
		    rev += strArr[i] + " ";
		}
		System.out.println(rev);
	}
}
